CardForm/build.gradle

```diff
@@ -14,6 +14,9 @@ android {
     testOptions {
         unitTests {
             includeAndroidResources true
+            all {
+                maxHeapSize = "1024m"
+            }
```

Comment:
Why did we need to increase unit test heap size? I didn't think CardForm has intensive unit testing.

Reply:
Travis was straight up 🤕 . It appears to run out of memory. We may revert some of this configuration, but it turns out the best solution was to replace the gradle `test` task with `testRelease` in the `travis.yaml` file.

The `test` task runs both `testDebug` and `testRelease` which may be too much for Travis to handle given the number of unit tests in this project. We also never deploy a debug artifact, and making the change allows the build to pass 🤷‍♂️ .